libs/novui/src/json-schema-components/JsonSchemaForm.tsx

```diff
@@ -46,18 +47,48 @@ export type JsonSchemaFormProps<TFormData = any> = JsxStyleProps &
   CoreProps &
   Pick<FormProps<TFormData>, 'onChange' | 'onSubmit' | 'onBlur' | 'schema' | 'formData' | 'tagName'> & {
     variables?: string[];
+    errors?: any;
   };
 
+const getExtraErrors = (errors: RJSFValidationError[]) => {
+  if (!errors) {
+    return {};
+  }
+  // adding a default error for the case of errors in an array not updating correctly
+  return { __errors: ['enableShowExtraErrors'], ...toErrorSchema(errors || []) };
+};
+
+// Always return an empty array to prevent the default error messages from showing
+function transformErrors(_: RJSFValidationError[]) {
+  return [];
+}
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Helpful comments, thank you!‏